### DISCUSSION TITLE
Canvas Panel Element Thresholds

### DISCUSSION THREAD

**Discussion by fadimki** [grafana/grafana#64945] [Dashboards, Panels & Visualizations]
Simply custom threshold for each element. This way being able to visualize the infrastructure and see which parts are red (down). Currently you can only set thresholds for the entire panel which makes it hard if you have different thresholds.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82825. Feel free to continue the discussion around this there. Thank you!
